rust/ql/test/query-tests/unusedentities/main.rs

```diff
@@ -111,7 +111,7 @@ fn arrays() {
 
     println!("lets use {:?}", js);
 
-    for k // SPURIOUS: unused variable [macros not yet supported]
+    for k
```

Comment:
I don't like the idea of changing formatting around just because query results change.  If it was correct to split the `for` loop into two lines before (so that any results on `k` or `ks` are clearer), it's still justified now IMO.

I'm admittedly not an unbiased source as changing this would require me to fix up a couple of other commits I have ready.  If you don't agree with my justification, say so and I'll make the change after all my work on this is merged.